images/bootstrap/create_bazel_cache_rcs.sh

```diff
@@ -101,8 +101,13 @@ make_bazel_rc () {
 # - The path specified by the --bazelrc=file startup option. If specified, this option must appear before the command name (e.g. build)
 # - A file named .bazelrc in your base workspace directory
 # - A file named .bazelrc in your home directory
-make_bazel_rc >> "${HOME}/.bazelrc"
+bazel_rc_contents=$(make_bazel_rc)
+echo "create_bazel_cache_rcs.sh: Configuring \`${HOME}/.bazelrc\` and \`/etc/bazel.bazelrc\` with"
```

Comment:
backticks in shell scripts make me a bit nervous since they can cause evaluation.

Reply:
these are escaped?

Reply:
switched to single quote